vue/components/ui/molecules/button-icon-toggle/button-icon-toggle.vue

```diff
@@ -74,7 +74,7 @@ export const ButtonIconToggle = {
             return this.valueData ? this.colorSecondary : this.color;
         },
         iconComputed() {
-            return this.valueData ? this.iconSecondary : this.icon;
+            return this.valueData ? this.iconSecondary || this.icon : this.icon;

```

Comment:
Move `this.iconSecondary || this.icon` to a separate `computed`, otherwise its meaning gets a little lost.